buildbot/buildbot/workproc.py

```diff
@@ -40,7 +40,7 @@ def start(self):
             if not thread.is_alive():
                 thread.start()
 
-    async def handle(self, client, addr):
+    async def handle(self, client, _):
```

Comment:
Not that it actually matters, but I’d leave the name of the unused parameter here in situations like this (and the config parameter above). Because these are both sort of callbacks from an existing API, I kind of like to know what information the API is providing to the function—even if the current version of the code doesn’t need it. It saves an extra trip to the docs to figure out what the unlabeled information is, and it makes it clear that the shape of the function matches all the other functions that play a similar role.